plugins/tcl.tk/TkTable.mk

```diff
@@ -0,0 +1,34 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := TkTable
```

Comment:
We do not use uppercase characters in package names. Can you change the name to tktable, please?

Reply:
done